ACMAS/Docker deployment instructions.md

```diff
@@ -0,0 +1,20 @@
+# Deployment instructions for ACMAS @ RCOS v2 (Django, Docker, Postgres, Gunicorn, Nginx)
+## Production server
+_Note: Running migrations and serving static files are now handled in entrypoint.prod.sh. and entrypoint.sh._
+
+[Docker](https://docs.docker.com/get-docker/) is a pre-requisite installation to run the deployment server.
+If you do not have Docker installed, please do so before continuing to the remainder of the instructions.
+### Build images and spin up production containers
+`docker-compose -f docker-compose.prod.yml up -d --build`
```

Comment:
also should have specifications on how to run the local host -> localhost:8000 or whatever is supposed to work

also could benefit from having some more explinations/being clear on what is actually happening so if users come across a problem, they at least know which step could be messing up

Reply:
Resolved